.github/workflows/build-release.yml

```diff
@@ -220,7 +231,7 @@ jobs:
 
       - name: Get Current Date
         id: date
-        run: echo "::set-output name=date::$(date +'%Y-%m-%d')"
+        run: echo "date=$(date + '%Y-%m-%d')" >> $GITHUB_OUTPUT
```

Comment:
see https://github.blog/changelog/2022-10-11-github-actions-deprecating-save-state-and-set-output-commands/